src/meta/src/manager/env.rs

```diff
@@ -56,13 +56,15 @@ where
 pub struct MetaOpts {
     pub enable_recovery: bool,
     pub checkpoint_interval: Duration,
+    pub in_flight_barrier_nums: usize,
 }
 
 impl Default for MetaOpts {
     fn default() -> Self {
         Self {
             enable_recovery: false,
             checkpoint_interval: Duration::from_millis(100),
+            in_flight_barrier_nums: 10,
```

Comment:
Can we reuse `default::in_flight_barrier_nums()` for this? Ditto for `checkpoint_interval`.

Reply:
sorry , what does "reuse" mean?

Reply:
Just call `default::in_flight_barrier_nums()` instead of putting a hard coded number here. :P